docs/source/examples/elasticsearch_outlet.rst

```diff
@@ -0,0 +1,85 @@
+ElasticSearch Outlet
+--------------------
+
+.. container:: tutorial-block
+
+    In this example we create an implementation of :any:`Outlet` that indexes documents to a running Elasticsearch instance. 
```

Comment:
Aren't we `indexing records as documents` rather than `indexing documents`? Just a guess, not even that sure of this. And this is semantics, so no worries if you ignore this comment

Reply:
Yeah this is a better way to put it. I defaulted to the generic document terminology of ES but yeah we are indexing records.